daemon/app/Main.hs

```diff
@@ -168,13 +173,14 @@ updateInbox chanMsg = incrementSN . updater
 
 webServer :: TVar ServerState -> IO ()
 webServer var = do
+  assets <- Assets.loadAssets
```

Comment:
load assets (like png files for web design) in the beginning of the server session. Assets are shared across multiple client sessions.